rules/linux/execution_find_binary.toml

```
@@ -24,9 +24,7 @@ timestamp_override = "event.ingested"
 type = "eql"
 
 query = '''
-sequence by host.id, process.pid with maxspan=1m
-[process where process.name == "find" and process.args : "-exec" and process.args : ("/bin/bash", "/bin/sh", "bash", "sh") and process.args : ";"]
-[process where process.parent.name == "find" and process.name : ("bash", "sh")]
+process where event.type == "start" and process.name in ("bash", "sh") and process.parent.name == "find" and process.parent.args == "-exec" and process.parent.args == ";" and process.parent.args in ("/bin/bash", "/bin/sh", "bash", "sh")
```

format only suggestion

```suggestion
process where event.type == "start" and process.name in ("bash", "sh") and 
  process.parent.name == "find" and process.parent.args == "-exec" and 
  process.parent.args == ";" and process.parent.args in ("/bin/bash", "/bin/sh", "bash", "sh")
```